getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Configure Image deployment by digest
+description: Specify the digests the operator will use to deploy images
+---
+
+{% assign operator = site.data.versions.first.tigera-operator %}
+
+### Big picture
+
+Enable deploying images by docker registry digest.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+{{site.prodname}} will not update existing tags but using an immutable digest allows specific images to be reviewed
+and verified by security teams.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+* **ImageSet**
+
+### Concepts
+
+#### Docker repository
+
+A docker respository provides access to docker images referenced by tags or digest.
+
+#### Image tag
+
+Versioned docker images are typically referenced by a tag which is appended to an image reference. Example: `<repo>/<image>:<tag>`. Docker image tags are typically not expected be changed or updated, but this is not required or enforced.
+
+#### Image digest
+
+Docker images, when added to a docker repository, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
```

Comment:
docker -> container

Reply:
(unless I am just off the mark and these are all docker-specific features?)

Reply:
I don't think it is a Docker (hub) registry specific feature, but I was thinking Docker Registry API feature. https://github.com/distribution/distribution/blob/main/docs/spec/api.md#digest-parameter
I'm not aware of other container registry APIs, so I think saying container is not wrong but I guess saying Docker is not perhaps not specific enough. Perhaps in the "Container repository" I should call it out that it implements the Docker Registry HTTP API. WDYT?